sdk/confidentialledger/Azure.Security.ConfidentialLedger/tests/samples/IdentityServiceClientSample.md

```diff
@@ -1,18 +1,18 @@
 # The Certificate Service Client 
 
-By default, the `ConfidentialLedgerClient` will automatically configure itself to trust the root TLS certificate advertised by the `ConfidentialLedgerCertificateClient` `GetLedgerIdentity` operation.
+By default, the `ConfidentialLedgerClient` will automatically configure itself to trust the root TLS certificate advertised by the `ConfidentialLedgerIdentityServiceClient` `GetLedgerIdentity` operation.
 However, if specific customizations are needed to the client option's `Transport`, this automatic configuration needs to be done manually.
 
 Because Confidential Ledgers use self-signed certificates securely generated and stored in an SGX enclave, the certificate for each Confidential Ledger must first be retrieved from the Confidential Ledger Identity Service.
 
 ```C# Snippet:GetIdentity
 Uri identityServiceEndpoint = new("https://identity.confidential-ledger.core.azure.com") // The hostname from the identityServiceUri
-var identityClient = new ConfidentialLedgerCertificateClient(identityServiceEndpoint);
+var identityClient = new ConfidentialLedgerIdentityServiceClient(identityServiceEndpoint);
```

Comment:
These snippets are generated; to update them, you'll need to change the associated test and then run the script to inject them.   The [contributing guide](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets) offers more detail around the steps needed.